cinema/views.py

```diff
@@ -35,8 +35,7 @@ class GenreViewSet(
 ):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
-    authentication_classes = (TokenAuthentication,)
-    permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
+    permission_classes = [IsAuthenticated]
```

Comment:
Why did you replace the last permission by this one? App should work in the same way, just another way of authentication had to be added